add documentation for reading lower-dimensional objects (#834)
radio-astro-tools/spectral-cube: docs/creating_reading.rst

          files. See the :doc:`big_data` page for more details about dealing
          with large data sets.

Reading images from file
------------------------

While spectral-cube is designed for cube analysis, in the course of normal
analysis procedures you are likely to need to load up one- and two-dimensional
subsets or views of the data.

You can load :class:`~spectral_cube.lower_dimensional_objects.Projection`
objects from FITS HDU objects with
:meth:`~spectral_cube.lower_dimensional_objects.Projection.from_hdu`.  Only

> FITS reading is supported::

Reviewer: ```suggestion
FITS reading is currently supported::
```
Raised a separate issue for loading 2D CASA images: https://github.com/radio-astro-tools/casa-formats-io/issues/47

    >>> from astropy.io import fits  # doctest: +SKIP
    >>> hdul = fits.open('file.fits')
    >>> projection = Projection.from_hdu(hdul)

> Note that if you pass in a :class:`astropy.io.fits.HDUList` object, data will be loaded

Reviewer: ```suggestion
Note that if you pass in a :class:`astropy.io.fits.HDUList` object, by default the data will be loaded
```

> from the first HDU; this can be overridden with the ``ext`` keyword.

Reviewer: ```suggestion
from the first HDU. To load a different HDU in the list, the index can be passed to the ``ext`` keyword (e.g., `ext=1` to load the second HDU in the list).
```
